### DISCUSSION TITLE
Corsair Commander Core error

### DISCUSSION THREAD

**Discussion by zerocool320** [liquidctl/liquidctl#486] [General]
Below is what gets detected on my PC (liquidctl list):

Device #0: Gigabyte RGB Fusion 2.0 8297 Controller
Device #1: Corsair Commander Core (experimental)
Device #2: Corsair Commander Pro

However, when I try to check the status of Commander Core (liquidctl --match Core status), I get the error message shown below.
ERROR: unexpected error with Corsair Commander Core (experimental): AssertionError('response does not match command')

My H100i Elite Capellix Liquid CPU Cooler is connected to the Commander Core.
I would appreciate any help with my issue and  thanks in advance.
I could not find a related discussion, my apologies if I missed any existing once.

Regards,
Barath Ramesh

**Comment by jonasmalacofilho** [comment]
This is a known issue with more recent firmware.  We're tracking it in issue #464, and you can subscribe it to get notified of updates.
